src/styles/calendar/calendar.scss

```
@@ -0,0 +1,31 @@
+.calendar {
+  display: flex;
+
+  margin: 0 auto;
+
+  flex-wrap: wrap;
+  max-width: 726px;
+  padding: 25px 10px;
+  gap: 1px;
+
+  $days:
+    "mon" 0,
+    "tue" ($squareSide + 1px),
```

you can create a variable for this calculation & just calculate 1 time this
like
$day-margin: $squareSide + $squareGap
```suggestion
    "mon" 0,
    "tue" $day-margin,
```